MNE/fiff/fiff_dig_point.cpp

```diff
@@ -82,3 +87,80 @@ FiffDigPoint::~FiffDigPoint()
 {
 
 }
+
+
+//*************************************************************************************************************
+
+QList<FiffDigPoint> FiffDigPoint::read(QFile &t_fileDig)
+{
+    QList<FiffDigPoint> dig;
```

Comment:
how about creating a FiffDigPointSet class fiff_dig_point_set.h/.cpp with a QList<FiffDigPoint> m_digs member - is it worth creating such a class? Maybe we add some dig point transformation functions to that class later on